tests/testthat/test-fit_mmrm.R

```diff
@@ -3,6 +3,7 @@ library(dplyr)
 # h_get_diagnostics ----
 
 test_that("h_get_diagnostics works as expected", {
+  testthat::skip_if_not(requireNamespace("TMB"))
```

Comment:
there is a dedicated function for this - also easier to follow

```suggestion
  testthat::skip_if_not_installed("TMB")
```

Reply:
TMB seem to be installed, but loading was off.

Reply:
in such case I would do skip-if-not-installed followed by explicit library to signal that the loading is what we are looking for

Reply:
tmb is in suggests though, it should implicitly included by mmrm package, and not by us. again, I think this is an image issue. i don't think this pr is good, and it is not solving the real issue

Reply:
> tmb is in suggests though, it should implicitly included by mmrm package

Then all is needed is just skip if not installed. Following nosuggest logic, the package should remain fully functional without suggested pacakge

> and not by us

Partially true - see above. I would say mmrm should specify the version - not here

Reply:
mmrm has specified it's version. we can ignore that line